lib/generators/htmx/install_generator.rb

```diff
@@ -40,38 +46,34 @@ def manifest(javascript_dir)
         Pathname.new(destination_root).join(javascript_dir, 'application.js')
       end
 
-      def setup_importmap
-        run 'bin/importmap pin htmx.org'
-
-        manifest = manifest('app/javascript')
-
+      def add_to_manifest(manifest, text)
         if manifest.exist?
-          append_file manifest, "\n#{IMPORTMAP_SETUP}"
+          append_file manifest, "\n#{text}"
         else
-          create_file manifest, IMPORTMAP_SETUP
+          create_file manifest, text
         end
       end
 
-      def setup_sprockets
-        manifest = manifest('app/assets/javascripts')
+      def setup_bun
+        run 'bun add htmx.org'
```

Comment:
I think this is installing the latest version of htmx. I'm wondering if we should fix the version here - and probably apply that to the rest of the JS bundling methods as well

Reply:
```suggestion
        run "bun add htmx.org@#{Htmx::Rails::HTMX_VERSION}"
```